src/NpmUtilities.js

```diff
@@ -152,16 +152,14 @@ export default class NpmUtilities {
     ChildProcessUtilities.exec("npm", ["publish", "--tag", tag.trim()], opts, callback);
   }
 
-  static getExecOpts(directory, registry) {
+  static getExecOpts(directory) {
     const opts = {
       cwd: directory,
     };
 
-    if (registry) {
-      opts.env = Object.assign({}, process.env, {
-        npm_config_registry: registry,
-      });
-    }
+    opts.env = Object.assign({}, process.env, {
+      npm_config_registry: "https://registry.npmjs.org/",
```

Comment:
We still need to respect the user's `--registry` argument, here, because not everyone actually _wants_ to use the default npm registry.

Reply:
Specifically, the `registry` should be defaulted to the public one _only_ in the [`publishTaggedInDir()` method](https://github.com/lerna/lerna/blob/a20601c6e94b3f651b6a8474dd1b1f00269f6be2/src/NpmUtilities.js#L151), not universally.